util/Hexagon/reports/cfgdot

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/python
```

Comment:
Please give Python files a `.py` extension

Reply:
Furthermore, these should all be `#!/usr/bin/env python` to allow virtual environments to work.